session/src/main/java/org/apache/iotdb/session/SessionConnection.java

```diff
@@ -800,6 +805,85 @@ protected void createSchemaTemplate(TSCreateSchemaTemplateReq request)
     }
   }
 
+  protected void createSchemaTemplate(TSCreateSchemaTemplateSerializedReq request)
```

Comment:
also add these methods to SessionPool

Reply:
methods at Session.java about template had been added to SessionPool.java with wrapper of retry like other methods